lib/price-import.coffee

```diff
@@ -134,6 +137,15 @@ class PriceImport extends ProductImport
     debug 'About to send %s requests', _.size(posts)
     Promise.settle(posts)
 
+  ###
+   * returns true if product does not have any staged changes
+   * and is published
+  ###
+  _canBePublished: (product) ->
+    if !product.hasStagedChanges and product.published
```

Comment:
@butenkor @PhilippSpo @mmoelli @emmenko @hajoeichler please look at the comment in https://github.com/sphereio/sphere-product-import/issues/66 for updates